pkgs/development/tools/continuous-integration/woodpecker/common.nix

```diff
@@ -1,9 +1,9 @@
 { lib, fetchFromGitHub }:
 let
-  version = "1.0.3";
-  srcHash = "sha256-P1ODGxHkrh8o+RMxxu7OCuHkShfLyQcF9KVNYw45T5A=";
-  vendorHash = "sha256-j2C66oTv0RY8VGDEivrj/p2PtGAhrDhi9oBvNXATurI=";
-  yarnHash = "sha256-TrcTc5svLLSedRC8gCwIBW7/mtHo+uSNZGImtRiVJ0w=";
+  version = "2.0.0-rc.0";
+  srcHash = "sha256-n4rJNBgOQV4rKoFirHW63q520HF8PYBmtwx7iEmht+E=";
+  vendorHash = "sha256-/GOwsrYzVoYjJZ27sp/gzp1N3de6P16v+kHY1u1wTKs=";
```

Comment:
Ah also, it looks like the update script is slightly broken. It doesn't update the `vendorHash` on the first run (I suspect because we haven't yet written the updated `version`/`srcHash` which are used implicitly in the `nix-prefetch` call).